plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveFailureRecovery.java

```diff
@@ -39,6 +42,17 @@ protected QueryRunner createQueryRunner(List<TpchTable<?>> requiredTpchTables, M
                 .build();
     }
 
+    @Override
+    protected void createLineitemTable(String tableName, List<String> columns, List<String> partitionColumns)
```

Comment:
Why not to simply create it in the `AbstractTestFailureRecovery`? (in a `@BeforeClass` method)

Reply:
The  `partitioned_by` table property can be different in each connector. That's the only reason to not do it in the abstract class directly.

Reply:
done